addons/point_of_sale/models/res_partner.py

```diff
@@ -110,3 +110,13 @@ def open_commercial_entity(self):
             **super().open_commercial_entity(),
             **({'target': 'new'} if self.env.context.get('target') == 'new' else {}),
         }
+
+    def _get_stat_info(self):
+        data_list = super()._get_stat_info()
+        if not self.env.user.has_group('point_of_sale.group_pos_user'):
+            return data_list
+        for partner in self:
```

Comment:
Same here, when you have a single "'if", use a filtered instead, you gain a line and an indent level :) (to propagate to other modules)